include/zephyr/bluetooth/gatt.h

```diff
@@ -172,7 +172,8 @@ struct bt_gatt_attr {
 	void *user_data;
 	/** Attribute handle */
 	uint16_t handle;
-	/** Attribute permissions */
+	/** Attribute permissions.
+	 * Will be 0 if returned from bt_gatt_discover() */
```

Comment:
The formatting fix for this should be for the right commit. You can use `git rebase` to modify this commit

Reply:
Should be fixed now, I hope. Sorry for all the mistakes, first time 😅

Reply:
Thank you for fixing it :)